compiler/src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -28,6 +28,7 @@ object Formatting {
       case arg: Showable =>
         try arg.show
         catch {
+          case ex: CyclicReference => "..."
```

Comment:
This would kick in if we hit a Cyclic Reference during printing. The idea is that printing should try to do the best it can rather than quitting with an exception. But I'll make clearer that we elided a cyclic reference.